app/controllers/application_controller.rb

```diff
@@ -15,11 +16,7 @@ def configure_permitted_parameters
   end
 
   def after_sign_in_path_for(resource)
-    if resource.class == User
```

Comment:
For reference, note that where multiple comparisons are required, the following is possible due to case using `===`:

```ruby
case obj
when User then x
when Admin then y
else
  z
end
```